types/index.d.ts

```diff
@@ -85,6 +85,5 @@ export interface Chalk {
 	bgCyanBright: Chalk;
 	bgWhiteBright: Chalk;
 }
-
-declare function chalk (): any;
-export default chalk as Chalk;
+declare const chalk: Chalk
```

Comment:
New line before the `declare` statement, please :)

Reply:
I've updated both PRs.